projects/example/project.yaml

```diff
@@ -0,0 +1,4 @@
+# Provide the home page for your project:
+homepage: "https://github.com/google/oss-fuzz"
+# Provide the e-mail for the primary contact:
+primary_contact: "kcc@google.com"
```

Comment:
We should probably add https://github.com/google/oss-fuzz/blob/master/projects/libchewing/project.yaml#L2, so that this example project is not fuzzed by ClusterFuzz, also can add a comment so that people don't add it.